src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -114,9 +114,9 @@ public boolean canCompileTo(Triplet target) {
         // if the host os and target os are the same, always return true
         if (getOs().equals(target.getOs())) return true;
 
-        // if host is linux and target is ios, fail
-        return (!Constants.OS_LINUX.equals(getOs()) && !Constants.OS_WINDOWS.equals(getOs())) ||
-                !Constants.OS_IOS.equals(target.getOs());
+        // so far, iOS can be built from Mac, Android can be built from Linux
+        return (OS_DARWIN.equals(getOs()) && Constants.OS_IOS.equals(target.getOs())) ||
```

Comment:
this will fail on Darwin as host and target

Reply:
if returns true before when host and target are the same.